nishimweelysee/Elysee-CapstoneProject
# Bg login and sign up

#### What does this PR do?
> This PR is for re-writing mu UI work on Feedback.
>
#### Description of Task to be completed
>- Modify Sign in Page
>- Modify Sign up page
>- Modify Landing page
#### How should this be manually tested?
>- Once you opened  my website you will see my image on landing page.
>- When you click on user profile image at top-right of page you can be able to login
>- When you are going to comment you will be asked to login
>- When you are going to Send Contact you will be asked to login
>- Once you logged in you will doing everything without be asked to login again.
>- When you are going to login and you do not have an account you will be the message tells you to sign up if you don't have an account.
#### Any background context you want to provide?

#### What are the relevant Trello story?
> BG( Update signin and Signup page) use this link to see my trello card :https://trello.com/c/jrunIYej
#### Screenshots (if appropriate)
>
>
![image](https://user-images.githubusercontent.com/56052588/94537811-8b6a5c80-0243-11eb-81dc-77a28710f043.png)
>
![image](https://user-images.githubusercontent.com/56052588/94537905-ab018500-0243-11eb-8e71-b937f9a3a36a.png)
>
![image](https://user-images.githubusercontent.com/56052588/94538004-c076af00-0243-11eb-9b1d-9038cc6b8583.png)





> Thank you !!!
